core/templates/pages/collection-player-page/collection-player-page.component.ts

```diff
@@ -283,13 +283,15 @@ export class CollectionPlayerPageComponent implements OnInit {
   }
 
   async fetchSummaryAsync(collectionId: string): Promise<void> {
-    let Summary = (
-      await
-      this.collectionPlayerBackendApiService.fetchCollectionSummariesAsync(
-        collectionId
-      ));
-    if (Summary) {
-      this.collectionSummary = Summary.summaries[0];
+    let summary = null;
+    this.collectionPlayerBackendApiService.fetchCollectionSummariesAsync(
+      collectionId
+    ).then((collectionSummary) => {
+      summary = collectionSummary;
+    });
+
+    if (summary) {
+      this.collectionSummary = summary.summaries[0];
```

Comment:
Just wondering why we are making this change?

Reply:
Earlier, there were http calls made from conversation-skin.directive.ts. They have to be moved to backend api service after migration.
I found that we already have backend api service which makes calls to same URL.
Few tweaks were required, so already exisiting backend api service function can be used for conversation.component.ts.

This change can be seen as side-effect of the changes made to make backend api service more generalized (so, it can be used at more than one place).